docs/changehistory/NextVersion.md

```diff
@@ -1553,7 +1277,7 @@ Several changes to the APIs for executing ECSql statements have been made to imp
   }
 ```
 
-### Id64Set parameter bindings
+##### Id64Set parameter bindings
```

Comment:
This is a new feature not a breaking change. @khanaffan

Reply:
thanks, moved.